lib/galaxy/tool_util/deps/container_resolvers/mulled.py

```diff
@@ -633,6 +633,14 @@ def pull(self, container):
             cmds = container.build_mulled_singularity_pull_command(
                 cache_directory=self.cache_directory.path, namespace=self.namespace
             )
+            # when Galaxy runs in Docker with uswgi, HOME remains set to /root, this is a workaround for that case
+            homedir = os.environ.get('HOME')
```

Comment:
Can you please run `make format` to ensure the `black` Python coding style is applied?